pkg/detector/policy.go

```diff
@@ -337,13 +337,3 @@ func (d *ResourceDetector) listCPPDerivedCRBs(policyID, policyName string) (*wor
 
 	return bindings, nil
 }
-
-// excludeClusterPolicy excludes cluster propagation policy.
-// If propagation policy was claimed, cluster propagation policy should not exist.
-func excludeClusterPolicy(objLabels map[string]string) bool {
```

Comment:
I suggest that this function should be reserved, you can remove residual annotations by this function.